packages/eslint-config/index.js

```diff
@@ -95,6 +104,7 @@ module.exports = {
       rules: {
         'import/no-extraneous-dependencies': 'off',
         '@typescript-eslint/no-non-null-assertion': 'off',
+        '@typescript-eslint/explicit-function-return-type': 'off',
```

Comment:
I'm dumb - why is this needed?

Reply:
Disabling in tests because there are a lot more offenders there. I figured we can be looser on return types in tests.